core/jazz_ui/src/app/pages/service-metrics/service-metrics.component.ts

```diff
@@ -92,24 +94,10 @@ export class ServiceMetricsComponent implements OnInit, AfterViewInit {
       return this.applyFilter();
     }
 
-    })
   }
 
   ngOnInit() {
     this.serviceType = this.service.type || this.service.serviceType;
-    this.setPeriodFilters();
-  }
-
-  setPeriodFilters() {
```

Comment:
@swati-sh Include this. It is used for GCP.

Reply:
Done